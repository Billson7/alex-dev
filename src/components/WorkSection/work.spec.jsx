import React from "react"
import { render } from "@testing-library/react"
import { Work } from "./"

test("renders Work section correctly", () => {
  const { container } = render(<Work />)

  expect(container).toMatchSnapshot()
})
