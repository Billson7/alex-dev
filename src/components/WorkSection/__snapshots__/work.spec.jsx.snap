// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Work section correctly 1`] = `
<div>
  <section
    class="section section-work"
  >
    <div
      class="flex-row"
    >
      <h2
        class="profile-heading"
      >
        Work
      </h2>
      <p
        class="profile-caption-invert"
      >
        typing, clicking, stackoverflow-ing
      </p>
      <p
        class="profile-intro"
      >
        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum
      </p>
      <p
        class="profile-intro"
      >
        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum
      </p>
      <code>
        github things? screenshots? links?
      </code>
    </div>
  </section>
</div>
`;
